src/Stage/Type.hs

```diff
@@ -0,0 +1,27 @@
+module Stage.Type where
```

Comment:
This module doesn't seem to buy us anything. Is it here only for consistency? I think I'd prefer to keep everything simply in `Stage`, as otherwise the latter looks really silly with a single `stageString` function.

Reply:
Got rid of this module and put back everything into `src/Stage.hs`, in an upcoming commit.